package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Compile LightScript to JavaScript.",
   "main": "index.js",
   "scripts": {
-    "test": "node node_modules/.bin/mocha test",
+    "test": "mocha test",
```

Comment:
can we make a note in the README that contributors will need to add `node_modules/.bin/` to their PATH?